api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md

```diff
@@ -1,10 +1,182 @@
 ---
 id: dxGantt.Options.sorting
 type: dxGanttSorting
+inheritsType: dxGanttSorting
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Configures sorting settings.
+
+---
+
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Gantt/Sorting/"
+}
+
+Users can sort Gantt data by a single or multiple columns. Use the [mode](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode) option to specify the sort mode. 
+
+- **Single Mode**. Click a column header to sort data by this column. Subsequent clicks on the same header reverse the sort order.
+
+    ![DevExtreme Gantt - Sorting](/images/Gantt/sorting-single.png)
+
+- **Multiple Mode**. Hold Shift and click a column header to sort data by multiple columns. Subsequent clicks on the same header reverse the sort order and clear sorting for the column. Set the [showSortIndexes](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#showSortIndexes) option to **true** to show a column sort index within the header.
```

Comment:
You write 'a column header', but 'by multiple columns' (singular vs plural). Maybe you can write as follows:
"Click column headers with the Shift key pressed to sort data by multiple columns. "

'Subsequent clicks ...... clear sorting for the column' - It's unclear how subsequent clicks can both reverse sort order and clear sorting, Maybe you can give more details here. Should I use 'Shift' for subsequent clicks?